cmo-metadb-sample-publisher/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
Do we need these checked in?  I'm wondering what they're for. I think only code/properties pertaining to the project itself should probably be checked in and not necessarily items which might be system/ide-related.